pyproject.toml

```
@@ -27,18 +27,17 @@ upstreaming = [
 name = 'pre-commit-terraform'
 classifiers = [
   'License :: OSI Approved :: MIT License',
-
   'Programming Language :: Python :: 3',
+  'Programming Language :: Python :: 3 :: Only',
   'Programming Language :: Python :: 3.9',
   'Programming Language :: Python :: 3.10',
   'Programming Language :: Python :: 3.11',
   'Programming Language :: Python :: 3.12',
   'Programming Language :: Python :: 3.13',
-
   'Programming Language :: Python :: Implementation :: CPython',
   'Programming Language :: Python :: Implementation :: PyPy',
 ]
-description = 'Pre-commit hooks for terraform_docs_replace'
+description = 'Pre-commit hooks for Terraform, OpenTofu, Terragrunt and related tools'
```

```suggestion
description = 'Pre-commit hooks for Terraform, OpenTofu, Terragrunt, and related tools'
```

I guess it's more common to add serial comma in this case rather than to not add to distinguish between "hooks for related tools also" and "hook for TF, Tofu, TF + some other tools related to this".

---

Hmm, the more I re-read the sentence the more ambiguous is whether to use or to not use serial comma 😕 I hate English grammar 🤣

---

There's many styles of English grammar. So you're good. If you want to enforce something specific, look into plugging Vale.

---

> If you want to enforce something specific, look into plugging Vale.

Nice to know. Thanks 👍🏻